build-scripts/components/k8s-dqlite/version.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-echo "v1.4.0"
+echo "fix-mem-leak"
```

Comment:
I think this one should be changed to :
```suggestion
echo "v1.4.1"
```